README.md

```diff
@@ -53,7 +53,82 @@ There are three steps to the CI workflow:
      doesn't update any services, only manifests, so the push from the `deploy`
      job will never run another `deploy` job.
 
-## WIP: deployment to kubernetes
+## WIP: Deploy slack service to kubernetes
+
+Create a kubernetes cluster somewhere
+
+```bash
+gcloud container clusters create dapr --num-nodes=1
+```
+
+Initialize Dapr on the cluster (https://docs.dapr.io/operations/hosting/kubernetes/kubernetes-deploy/)
+
+```bash
+dapr init -k
+```
+
+Create the slack Node App. 
+
+```bash
+kubectl apply -k manifests/slack/
+```
+
+Create the secret to authenticate with the GitHub Container registry
```

Comment:
We could use `kubeseal` to create a [sealed secret](https://github.com/bitnami-labs/sealed-secrets) using a PAT from a [Red Badger bot](https://github.com/badgerbot).
That way we could also apply it to any new cluster

Reply:
Hi David, I might create an issue for this if that's ok. I haven't used kubeseal before, but it looks like the ceritificate is specific to a cluster, so the sealing process would need to be done / redone for each cluster we wanted to deploy to. I expect there are other options as well which might be worth exploring (although possible red badger has already done this exploring and come to a conclusion). Azure has a Key Vault for storing secrets for example, and its own container registry, which is more convenient to work with than an external one.

Reply:
Yeah good idea, also using GCR and kubernetes on GCP means you don't need image pull secrets.

Reply:
https://github.com/redbadger/badger-brian/pull/11